docs/source/management/index.md

```diff
@@ -108,16 +112,104 @@ Available SRE configurations for context 'green':
 └──────────────┴──────────┘
 ```
 
-## Removing a deployed Data Safe Haven
+### Removing a deployed Data Safe Haven
 
 - Run the following if you want to teardown a deployed SRE:
 
 ```{code} shell
 $ dsh sre teardown YOUR_SRE_NAME
 ```
 
+::::{admonition} Tearing down an SRE deletes ALL data and resources
+Running `dsh sre teardown` will remove all resources deployed within the SRE.
+Ensure that any desired outputs have been egressed before deleting the SRE.
+Any data remaining on the SRE will be deleted.
+The user groups for the SRE on Microsoft Entra ID will also be deleted.
+::::
+
 - Run the following if you want to teardown the deployed SHM:
 
 ```{code} shell
 $ dsh shm teardown
 ```
+
+::::{admonition} Tearing down an SHM
+All SREs associated with the SHM must be torn down before the SHM can be torn down.
```

Comment:
Is this true? As in, if I run `dsh shm teardown` now will it stop me if there are SREs?

Reply:
Depends if we want to merge #2256 (if I finish it in time)

Reply:
Let's bump that to 5.0.2